sdk/lib/opentelemetry/sdk/trace/span.rb

```diff
@@ -159,7 +159,7 @@ def record_exception(exception, attributes: nil)
           event_attributes = {
             'exception.type' => exception.class.to_s,
             'exception.message' => exception.message,
-            'exception.stacktrace' => exception.full_message(highlight: false, order: :top)
+            'exception.stacktrace' => exception.full_message(highlight: false, order: :top).encode('UTF-8', invalid: :replace, undef: :replace, replace: '?')
```

Comment:
This is a bit of an odd edge case.  If your stacktrace contains something that cannot be encoded in UTF-8 it will cause exceptions during export.  This pre-emptively handles that.